### DISCUSSION TITLE
Cannot hit notes on certain songs on Kade Engine 1.5.4

### DISCUSSION THREAD

**Discussion by ACrazyTown** [KadeArchive/Kade-Engine#2757] [Q&A]
I'm having an issue with KE1.5.4 where on some songs there are notes that are literally UNHITTABLE, no matter what I do, it's always the same notes and I cannot hit them for the love of god.
I don't know if it's the input system or something causing this, but I'd like to know if there's any solution.
I'm also not the only one experiencing this issue, I've talked to some people on Discord and they've said they also have the same issue on KE1.5.4

Anyone know how to fix this?

**Comment by Lumaah** [comment]
You have the redo the part where you can't hit notes with the chart editor

**Reply by ACrazyTown** [reply]
Nah, wasn't that. Someone already answered before but I guess their comment got deleted, this is a known bug and was fixed in a later update

**Reply by ACrazyTown** [reply]
> Nah, wasn't that. Someone already answered before but I guess their comment got deleted, this is a known bug and was fixed in a later update

Nvm, mistaked this thread for another one, but it's still the same thing.
I needed to copy some code from a later commit
